src/ru/comx/src/eu/kanade/tachiyomi/extension/ru/comx/ComX.kt

```diff
@@ -108,7 +108,6 @@ class ComX : ParsedHttpSource() {
         .build()
 
     override fun headersBuilder(): Headers.Builder = Headers.Builder()
```

Comment:
```suggestion
    override fun headersBuilder(): Headers.Builder = super.headersBuilder()
```
so default useragent is preserved from super class